contrib/fixclock-hook

```diff
@@ -26,27 +28,96 @@ You can change the NTP server like this:
 Unfortunately, something went wrong...
 
 $ERROR
+"""
+
+LXC_TPL = f"""\
+##########################
+## FIXCLOCK HOOK FAILED ##
+##########################
+
+Amazon S3 and Duplicity need a UTC synchronized clock, but TKLBAM detected
+that we're running in a container ($CONTAINER).
 
+As the container host manages the local time (and an unprivileged container
```

Comment:
if the container has the required privileges, it can change the host clock. i think it is still best to avoid doing that even if we can — principle of least surprise. but then maybe reword the message a bit to clarify that this hook doesn't run inside containers as a matter of principle

Reply:
Ok thanks man. I'll tweak the text a little to make it correct.